resources/quota_resource.go

```diff
@@ -71,6 +71,7 @@ func (al *AppLimit) UnmarshalJSON(rawJSON []byte) error {
 type ServiceLimit struct {
 	TotalServiceInstances *types.NullInt `json:"total_service_instances,omitempty"`
 	PaidServicePlans      *bool          `json:"paid_services_allowed,omitempty"`
+	TotalServiceKeys      *types.NullInt `json:"total_service_keys,omitempty"`
 }
 
 func (sl *ServiceLimit) UnmarshalJSON(rawJSON []byte) error {
```

Comment:
**question** Do we need some custom logic + test to handle the nil case? Seems like we have that already for `TotalServiceInstances`. See https://github.com/cloudfoundry/cli/pull/2881/files#diff-305a3cec50fa59b749c31f4599960a65a465a66bfca7afe1ccce4fcb58fad06eL87-L92